### ISSUE TITLE
OIDC Application config is confusing to our customers

### ISSUE THREAD

**Issue by fgerschwiler** [caos/zitadel#3077]
While watching a customer changing an oidc application configuration, we noted that the whole form is somehow confusing.
They had to add a new redirect url, but this is not saved after clicking the "+" button. The user has to click the save button below the oidc configuration part.
In my opinion this save button is to far away, so the user don't get that the save button is also for the fields in the top.
Also it is not really clear, that the big buttons (pkce, code, etc) change the oidc configuration.

Maybe it makes sense, to redesign the whole page, so the users will not be confused anymore.

**Acceptance criteria**
- [ ] ui design for detail page of oidc applications

**Comment by livio-a** [comment]
> Another issue has been raised over on Discord: Some applications seem to give you the redirect_url only after you have configured the IDP. The Wizard in ZITADEL doesn't like this and complains about missing redirect_url
> 
> @juergrinaldi for more info

I just tested: The wizard lets you continue without any warning. But on the created app you'll get a warning that your app is not compliant

**Comment by juergrinaldi** [comment]
I see a couple of issues in this that can be handled without re-designing the whole page, imo.

1. The saving problem should be handled when/if we transition to the 'auto-save principle' described in #3022 
2. To mitigate the issue with the changed oidc config when selecting one of the other 'boxes/buttons' we could inform the user via toast message that the oidc configs have changed. I'd prefer this way over the re-design because those buttons were designed as some preset guides for non-advanced users. deviate from this approach would probably make it harder again for the standard user to set up OIDC.
